### DISCUSSION TITLE
keep bluethooth pairing information

### DISCUSSION THREAD

**Discussion by msei99** [ltsp/ltsp#305] [Q&A]
Is there a way, that the bluethooth pairing information get saved between reboots?

I can pair my apple airpods and everything is working fine. After every reboot I have to pair my airpods again.

**Comment by alkisg** [comment]
That information might be saved somewhere under /var. You'd need to find out the exact dir, for example let's say it's /var/bluetooth.
LTSP currently doesn't save "per machine" state.
So the solution would be to mount /var/bluetooth from the server via e.g. NFS, with an FSTAB_VAR_BLUETOOTH="..." ltsp.conf line.

**Comment by msei99** [comment]
Thank you. It is working with this line in ltsp.conf (replace "user" with a existing username)
[MAC-ADR-OF-CLIENT]
FSTAB_VAR_BLUETOOTH="server:/home/user/.bluethooth /var/lib/bluetooth nfs   rw	0	0"

and adding a nfs-export to: /etc/exports.d/ltsp-nfs.exports
/home/user/.bluethooth *(rw,async,no_subtree_check,no_root_squash,insecure)

Don't forget to restart the nfs-server and do a "ltsp initrd"

Now the pairing information get saved to the homedirectory of the user under .bluethooth
